scripts/article1_post_draw.gml

```diff
@@ -8,16 +8,23 @@ if (point_distance(player_id.x, owner_true_y, tethered_id.x, tethered_true_y) >=
 
 if (x_dist == 0) {
     // we divide by x_dist so we need a special case when its 0
-    draw_debug_text(floor(view_get_xview() + 16), floor(view_get_yview() + view_get_hview() - 16), "lmao");
+    debug_draw(1, "X positions are equal");
 } else {
     target_val = power(sqrt(sqr(chain_length) - sqr(y_dist)) / abs_x_dist - 1, -0.5);
     guess = target_val / (2 * sqrt(6));
+    if (guess < 0.05) {
+        guess = 0.05;
+    }
     error = error_calc(guess, target_val);
-    for (i = 0; i < 4; i++) {
+    for (i = 0; i < 3; i++) {
```

Comment:
lowered precision here since the higher precision was mainly for the low value cases anyway but they are working fine now with the guess change